py-json/interop_connectivity.py

```diff
@@ -617,41 +617,41 @@ def __init__(self,
         self.mac = 'xx:xx:xx:*:*:xx'
 
     # set encoding value
-    def set_encoding(self, encryption, ieee80211u, ieee80211w, enable_pkc, bss_transition, power_save, disable_ofdma,
-                     roam_ft_ds, key_management, pairwise, private_key, ca_cert, client_cert, pk_passwd, pac_file):
+    def set_encoding(self, encryption, ieee80211u=True, ieee80211w=1, enable_pkc=True, bss_transition=True, power_save=True, disable_ofdma=True,
```

Comment:
Can you please re-submit this patch with Verified CLI, as per scripting patch requirements in Confluence. I  understand this script is included by other scripts, so a CLI from one of those other scripts verifying these changes would be great.